http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -201,13 +201,33 @@ class SamAuthProvider[F[_]: OpenTelemetryMetrics](
     else
       samDao.createResourceWithParent(samResource, creatorEmail, googleProject)
 
+  override def notifyResourceCreatedV2[R](
+    samResource: R,
+    creatorEmail: WorkbenchEmail,
+    cloudContext: CloudContext,
+    workspaceId: WorkspaceId,
+    userInfo: UserInfo
+  )(implicit sr: SamResource[R], encoder: Encoder[R], ev: Ask[F, TraceId]): F[Unit] =
+    if (sr.resourceType != SamResourceType.App)
```

Comment:
Re my other comment, maybe we only call `createResourceWithParentV2` if the resourceType is App and the cloud is GCP. Otherwise just `createResourceV2`